### DISCUSSION TITLE
Outrun Sega Enhanced

### DISCUSSION THREAD

**Discussion by LodanZark** [djyt/cannonball#87] [General]
This might be considered off-topic, but which version it's the latest, the outrun sega enhanced the 2.02 (https://reassembler.blogspot.com/2020/04/outrun-enhanced-edition-201.html) or the  2.0.3 (https://reassembler.blogspot.com/2011/08/outrun-enhanced-edition.html) ?

**Comment by djyt** [comment]
OutRun Enhanced Edition is the latest release, and a complete rewrite from the previous version.
https://reassembler.blogspot.com/2020/04/outrun-enhanced-edition-201.html

**Reply by LodanZark** [reply]
Thanks for the reply!
